PowerballApi/App_Start/Cacher.cs

```diff
@@ -1,20 +1,21 @@
-﻿namespace PowerballApi.Api
+﻿using System.Collections.Generic;
+using PowerballApi.Api.Models;
+
+namespace PowerballApi.Api
 {
 	using System;
 	using System.Runtime.Caching;
 
 	public class Cacher
     {
-        public void CacheData()
+        public void CacheData(PowerballSetList powerballSetList)
```

Comment:
I don't think you'll need this new object. Just pass in / cache a list of powerball sets.